website/integrations: Wiki.js: add instructions for group mappings (#23491)
goauthentik/authentik: website/integrations/documentation/wiki-js/index.md

In Wiki.js, configure the authentication strategy with these settings:

- **Client ID**: Client ID from the authentik provider.
- **Client Secret**: Client Secret from the authentik provider.
- **Authorization Endpoint URL**: https://authentik.company/application/o/authorize/
- **Token Endpoint URL**: https://authentik.company/application/o/token/

> - **User Info Endpoint URL**: https://authentik.company/application/o/userinfo/

Reviewer: ```suggestion
- **Authorization Endpoint URL**: `https://authentik.company/application/o/authorize/`
- **Token Endpoint URL**: `https://authentik.company/application/o/token/`
- **User Info Endpoint URL**: `https://authentik.company/application/o/userinfo/`
```

- **Issuer**: `https://authentik.company/application/o/<application_slug>/`
- **Logout URL**: `https://authentik.company/application/o/<application_slug>/end-session/`
- **Allow self-registration**: Enabled
- **Assign to group**: The group to which new users logging in from authentik should be assigned. (Please note that this takes prescendence over **Map Groups** in recent versions of Wiki.js.)

![](./wiki-js_strategy.png)

:::info
You do not have to enable "Allow self-registration" and select a group to which new users should be assigned, but if you don't, you will have to manually provision users in Wiki.js and ensure that their email addresses match the ones they have in authentik.
:::

> In recent versions of Wiki.js, there is a **Map Groups** feature, which will assign users to Wiki.js groups seen in the token received from authentik. To use this feature, the value of **Groups Claim** (e.g. `wiki-groups`) must be included in the `profile` claim from authentik. Here's how to make that happen:

Reviewer: ```suggestion
Wiki.js has a **Map Groups** feature, which will assign users to Wiki.js groups seen in the token received from authentik. To use this feature, the value of **Groups Claim** (e.g. `wiki-groups`) must be included in the `profile` claim from authentik. To set this up:
```
